src/AWSSQS.jl

```diff
@@ -20,11 +20,11 @@ export sqs_list_queues, sqs_get_queue, sqs_create_queue, sqs_delete_queue,
 
 
 using AWSCore
-using SymDict
+using AWSSDK.SQS
 using Retry
-using MbedTLS
+using SymDict
 using HTTP
-
+using MbedTLS
```

Comment:
any reason for changing the order of HTTP and MbedTLS?

Reply:
Nope, just the SymDict was needed. Otherwise it won't compile.

Reply:
Are you saying the order of `using SymDict` was an issue? What was the error message?